projects/plugins/jetpack/modules/likes.php

```diff
@@ -40,6 +40,12 @@
  * Jetpack Like Class
  */
 class Jetpack_Likes {
+	/**
+	 * Determines whether the Jetpack class and method exists. Default is true.
```

Comment:
Does it make sense to say "determines whether we're running/executing inside the Jetpack plugin"? I think it conveys our intention better.